### DISCUSSION TITLE
Problema ao compilar para linux

### DISCUSSION THREAD

**Discussion by redesoftware** [HashLoad/horse#370] [General]
Bom dia,

Desenvolvemos uma aplicação para Linux NOGUI/CONSOLE que está apresentando os erros abaixo após finalizar a compilação (F9), quando começa a fazer o linking:

[DCC Error] E2597 c:\program files (x86)\embarcadero\studio\22.0\lib\Linux64\debug\IdHTTPWebBrokerBridge.o(.data+0x50): error: undefined reference to '__wd_init_record_Web.WebBroker'
[DCC Fatal Error] F2588 Linker error code: 1 ($00000001)

Caso eu remova o horse da aplicação de teste ela compila e executa normalmente. Estou usando o Delphi 11.3, compilando no linux Ubuntu 22.04.3 LTS.

Alguém já teve este problema?

**Comment by viniciussanchez** [comment]
Qual a diretiva você está usando do Horse? HORSE_DAEMON?

**Comment by redesoftware** [comment]
Estou com o mesmo problema. Ao remover o Horse compila normalmente.

**Comment by redesoftware** [comment]
Consegui resolver o problema adicionando ao library path "$(BDS)\source\Indy10\Protocols".

**Comment by moraiscg** [comment]
"$(BDS)\source\Indy10\Protocols"   Realmente resolve
